packages/map-template/src/helpers/SetMapZoomLevel.js

```diff
@@ -0,0 +1,12 @@
+const mapsindoors = window.mapsindoors;
+
+/**
+ * Check if the solution allows the zoom level to be 22.
+ * If yes, set the zoom level to 22, otherwise set it to 21.
+ */
+export default function SetMapZoomLevel(mapsIndoorsInstance) {
```

Comment:
```suggestion
export default function setMapZoomLevel(mapsIndoorsInstance) {
```

It's usually only React components that has a capital first letter.